cmd/statusd/main.go

```diff
@@ -1,174 +1,134 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log"
 	"os"
-	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/status-im/status-go/geth/api"
 	"github.com/status-im/status-go/geth/params"
-	"gopkg.in/urfave/cli.v1"
 )
 
 var (
-	gitCommit  = "rely on linker: -ldflags -X main.GitCommit"
-	buildStamp = "rely on linker: -ldflags -X main.buildStamp"
-	app        = makeApp(gitCommit)
-	statusAPI  = api.NewStatusAPI()
+	gitCommit  = "N/A" // rely on linker: -ldflags -X main.GitCommit"
+	buildStamp = "N/A" // rely on linker: -ldflags -X main.buildStamp"
 )
 
 var (
-	// ProdModeFlag is whether we need dev or production settings
-	ProdModeFlag = cli.BoolFlag{
-		Name:  "production",
-		Usage: "Whether production settings should be loaded",
-	}
+	prodMode       = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile    = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir        = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID      = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
+	whisperEnabled = flag.Bool("shh", false, "SHH protocol enabled")
+	swarmEnabled   = flag.Bool("swarm", false, "Swarm protocol enabled")
+	httpEnabled    = flag.Bool("http", false, "HTTP RPC enpoint enabled (default: false)")
+	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
+	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC enpoint enabled")
+	logLevel       = flag.String("log", "", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
```

Comment:
Maybe WARN or ERROR by default?

Reply:
Oh, right. Originially we decided to switch off logging here, because the same code supposed to be used on the mobile, but now, as mobile-related code resides in `lib/` it makes perfect thanks 👍